literature.bib

```diff
@@ -1738,6 +1738,35 @@ @proceedings{VaMoS23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2022 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{BZD+:SANER22,
+  author = {John Businge and Ahmed Zerouali and Alexandre Decan and Tom Mens and Serge Demeyer and Coen De Roover},
+  title = {{Variant Forks - Motivations and Impediments}},
+  booktitle = SANER,
+  pages = {867--877},
+  publisher = IEEE,
+  address = Piscataway,
+  year = 2022,
+  doi = {10.1109/SANER53432.2022.00105},
+}
+
+% LAD+:PACMHCI22
+@article{LAD+:EICS22,
+  author = {Leclercq, Tony and Abbasi, Ebrahim Khalil and Dumas, Bruno and Remiche, Marie-Ange and Heymans, Patrick},
+  title = {{Essential Expectations of Users of Web Configurators: An Empirical Survey}},
+  year = 2022,
+  issue_date = {June 2022},
```

Comment:
do we usually track this? it follows from the year and month fields

Reply:
We don't, I will remove this.